connectors-e2e-test/connectors-e2e-test-agentic-ai/src/test/java/io/camunda/connector/e2e/agenticai/a2a/A2aStandaloneTests.java

```diff
@@ -0,0 +1,392 @@
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.web.server.LocalServerPort;
+import org.springframework.core.io.Resource;
+import org.springframework.test.context.TestPropertySource;
+
+@SlowTest
+@TestPropertySource(
+    properties = {
+      "camunda.connector.polling.enabled=true",
+      "camunda.connector.webhook.enabled=true"
+    })
+@WireMockTest
+public class A2aStandaloneTests extends BaseAgenticAiTest {
+
+  @Value("classpath:a2a-connectors-standalone.bpmn")
+  protected Resource testProcess;
+
+  @LocalServerPort private int port;
+
+  private WireMockRuntimeInfo wireMock;
+  private String webhookUrl;
+
+  @BeforeEach
+  void setUp(WireMockRuntimeInfo wireMock) {
+    this.wireMock = wireMock;
+    WireMock.reset();
+    webhookUrl = "http://localhost:%s/inbound/test-webhook-id".formatted(port);
+    setUpWireMockStubs();
+  }
+
+  @Test
+  void executeStandaloneA2aConnectorsPolling() throws IOException {
+    BpmnModelInstance bpmnModel = Bpmn.readModelFromStream(testProcess.getInputStream());
+    ZeebeTest zeebeTest =
+        createProcessInstance(
+                bpmnModel,
+                Map.of(
+                    "responseRetrievalMode", "polling", "a2aServerUrl", wireMock.getHttpBaseUrl()))
+            .waitForProcessCompletion();
+
+    CamundaAssert.assertThat(zeebeTest.getProcessInstanceEvent())
+        .hasVariableSatisfies(
+            "travelAgentFetchCardResponse",
+            A2aClientResponse.class,
+            A2aStandaloneTests::assertAgentCard)
+        .hasVariableSatisfies(
+            "travelAgentPollingResponse", A2aTask.class, A2aStandaloneTests::assertTask);
+  }
+
+  @Test
+  void executeStandaloneA2aConnectorsWebhook() throws Exception {
+    BpmnModelInstance bpmnModel = Bpmn.readModelFromStream(testProcess.getInputStream());
+    ZeebeTest zeebeTest =
+        createProcessInstance(
+            bpmnModel,
+            Map.of(
+                "responseRetrievalMode",
+                "webhook",
+                "a2aServerUrl",
+                wireMock.getHttpBaseUrl(),
+                "webhookUrl",
+                webhookUrl));
+
+    // Post working state - should NOT activate webhook
+    callWebhookEndpointWithDelay(
+        webhookUrl,
+        extractTaskFromJsonRpc(testFileContent("travel-agent-response-working.json").get()),
+        5);
+
+    // Post completed state - should activate webhook
+    callWebhookEndpointWithDelay(
+        webhookUrl,
+        extractTaskFromJsonRpc(testFileContent("travel-agent-response-completed.json").get()),
+        10);
+
+    zeebeTest.waitForProcessCompletion();
+
+    assertVariablesWithWebhook(zeebeTest);
+  }
+
+  @Test
+  void executeStandaloneA2aConnectorsWebhookWithToken() throws Exception {
+    var token = "test-token-123";
+    BpmnModelInstance bpmnModel =
+        BpmnFile.replace(
+            testProcess.getFile(), replace("id=\"A2A_Standalone\"", "id=\"A2A_Standalone_Token\""));
```

Comment:
So that the connector runtime replaces the already registered A2A webhook with a new one for this process.